README.md

```diff
@@ -80,20 +80,13 @@ docker exec -it $(docker ps --filter name=jobmanager --format={{.ID}}) /bin/sh
 Contributions to FlinkFood are welcome! Follow these steps to contribute:
 
 1. Create a new branch for your feature from the develop branch (`git checkout -b "(feature | hotfix | release | docs) / #of_task - really_brief_description"`) no spaces.
-2. Make changes, re-run the docker containers to test and commit (`git commit -m 'name_of_created_branch'`). In case you have done the work along-side someone (Parallel computing), you can commit with them as co-authors (how to do it: [here](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors))
+2. Make changes, re-run the docker containers to test and commit (`git commit -m 'number_of_task/simple_commit_message'`, eg: `git commit -m '91/update_README.md'`). In case you have done the work along-side someone (Parallel computing), you can commit with them as co-authors (how to do it: [here](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors))
```

Comment:
As I said [here](https://discord.com/channels/1161389681842343997/1174250586758201355/1175111183251017799) this does not work